xchange-core/src/main/java/org/knowm/xchange/dto/account/FundingRecord.java

```diff
@@ -242,32 +242,48 @@ public static Type fromString(String ledgerTypeString) {
     /** The user has requested the withdrawal or deposit, or the exchange has detected an initiated deposit,
      * but the exchange still has to fully process the funding.
      * The funds are not available to the user. The funding request may possibly still be cancelled though. */
-    PROCESSING,
+    PROCESSING("WAIT CONFIFMATION","EMAIL CONFIRMATION","VERIFYING","PENDING_APPROVAL","PENDING"),
```

Comment:
"CONFI**F**MATION", are you sure? It's "Wait Confi**r**mation" in `OkCoinRecords.java`.